test/cypress/integration/stripe-setup/vat-number-company-number/index_spec.js

```diff
@@ -0,0 +1,180 @@
+'use strict'
+
+// Local dependencies
+const commonStubs = require('../../../utils/common_stubs')
+const {
+  stubGetGatewayAccountStripeSetupSuccess,
+  stubGetStripeAccountSuccess
+} = require('./support')
+
+describe('Stripe setup: VAT number company number index page', () => {
+  const gatewayAccountId = 42
+  const userExternalId = 'userExternalId'
+
+  describe('Card gateway account', () => {
+    describe('when user is admin, account is Stripe and VAT number company number is not finished', () => {
```

Comment:
I think you want some words between “VAT number” and “company number”

Reply:
As discussed the naming was indicating that it is "the flag / the functionality", so I renamed it to `"VAT number / company number"`